app/views/merge_requests/merging_organisations.html.erb

```diff
@@ -38,7 +38,10 @@
     <%= form_with model: @merge_request, url: merge_request_path(id: @merge_request.id), method: :patch do |f| %>
       <% if @merge_request.merging_organisations.count.positive? %>
         <%= f.hidden_field :page, value: "merging_organisations" %>
-        <%= f.govuk_submit submit_merge_request_button_text(request.query_parameters["referrer"]) %>
+        <div class="govuk-button-group">
+          <%= f.govuk_submit submit_merge_request_button_text(request.query_parameters["referrer"]) %>
+          <%= govuk_link_to(secondary_merge_request_link_text(request.query_parameters["referrer"]), merge_request_path(@merge_request)) %>
+        </div>
```

Comment:
Added to absorbing org, merge date and merging orgs questions in addition to helpdesk ticket question